lib/ansible/plugins/connection/grpc.py

```diff
@@ -0,0 +1,270 @@
+      - section: grpc
+        key: root_certificates_file
+    env:
+      - name: ANSIBLE_ROOT_CERTIFICATES_FILE
+    vars:
+      - name: ansible_root_certificates_file
+  certificate_chain_file:
+    description:
+      - The PEM encoded certificate chain file used to create a SSL-enabled
+        channel, if the value is None in that case no certificate chain is used.
+    ini:
+      - section: grpc
+        key: certificate_chain_file
+    env:
+      - name: ANSIBLE_CERTIFICATE_CHAIN_FILE
+    vars:
+      - name: ansible_certificate_chain_file
+  grpc_channel_options:
+    description:
+      - The options provides a way to configure the gRPC channel with the optional
+        channel args defined by gRPC core runtime. The value of this option is of
+        type dict and the key-value pairs within dictionary should have valid values.
+    type: dict
+    ini:
+      - section: grpc
+        key: channel_options
+    env:
+      - name: ANSIBLE_GPRC_CHANNEL_OPTIONS
+    vars:
+      - name: ansible_grpc_channel_options
+  persistent_connect_timeout:
+    type: int
+    description:
+      - Configures, in seconds, the amount of time to wait when trying to
+        initially establish a persistent connection. If this value expires
+        before the connection to the remote device is completed, the connection
+        will fail.
+    default: 30
+    ini:
+      - section: persistent_connection
+        key: connect_timeout
+    env:
+      - name: ANSIBLE_PERSISTENT_CONNECT_TIMEOUT
+    vars:
+      - name: ansible_connect_timeout
+  persistent_command_timeout:
+    type: int
+    description:
+      - Configures, in seconds, this timeout is used as the default timeout
+        value when awaiting a response after issuing a call to a RPC. If the RPC
+        does not return in timeout seconds, an error is generated and connection is
+        closed.
+    default: 30
+    ini:
+      - section: persistent_connection
+        key: command_timeout
+    env:
+      - name: ANSIBLE_PERSISTENT_COMMAND_TIMEOUT
+    vars:
+      - name: ansible_command_timeout
+  persistent_log_messages:
+    type: boolean
+    description:
+      - This flag will enable logging the command executed and response received from
+        target device in the ansible log file. For this option to work 'log_path' ansible
+        configuration option is required to be set to a file path with write access.
+      - Be sure to fully understand the security implications of enabling this
+        option as it could create a security vulnerability by logging sensitive information in log file.
+    default: False
+    ini:
+      - section: persistent_connection
+        key: log_messages
+    env:
+      - name: ANSIBLE_PERSISTENT_LOG_MESSAGES
+    vars:
+      - name: ansible_persistent_log_messages
+"""
+
+from ansible.errors import AnsibleConnectionFailure, AnsibleError
+from ansible.module_utils.six import iteritems
+from ansible.plugins.loader import grpc_loader
```

Comment:
Could not find grpc_loader in here. Am I missing something? Or does this need to be added?

Reply:
Fixed